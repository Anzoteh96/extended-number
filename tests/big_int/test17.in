-5338818761877259700806897219282125945825935139799378
-138320248399982343877285528707397046215156693437429
