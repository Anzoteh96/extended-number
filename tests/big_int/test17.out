Enter two numbers n and m:
n is -5338818761877259700806897219282125945825935139799378
m is -138320248399982343877285528707397046215156693437429
n + m is -5477139010277242044684182747989522992041091833236807
n - m is -5200498513477277356929611690574728899610778446361949
n * m is 738466737305348749287709677216298453368061798854027614597916073228425987130027458505807836204256119162
n / m is 38
n % m is -82649322677930633470047128401038189649980789177076
